.env.example

```
@@ -172,3 +172,44 @@ APPLE_API_ISSUER_ID=
+
+# --- SKU deploy script (scripts/sku/) ---
+# Used only by `pnpm sku:diff` / `pnpm sku:apply`. The runtime never reads
+# these. Both blocks are optional; if unset the script falls back to the
+# matching runtime credential — works only if the runtime credential has
+# already been elevated to write scope.
+
+# Apple App Store Connect REST API. Requires App Manager role (or a key
+# scoped for in-app purchase write). Different from the runtime "In-App
+# Purchase" view-only key.
+APPLE_CONNECT_API_KEY_ID=
+APPLE_CONNECT_API_ISSUER_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APPLE_CONNECT_API_ISSUER_ID key should go before the APPLE_CONNECT_API_KEY_ID key

(UnorderedKey)